Directory.Build.props

```diff
@@ -8,10 +8,16 @@
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <OutputDrop>$(TF_BUILD_BINARIESDIRECTORY)</OutputDrop>
+    <NoWarn>$(NoWarn),1570,1572,1573,1574,1591,1701</NoWarn>
```

Comment:
I see you added 1701 because of all those redirect warnings... is it possible that we may miss one that results in a runtime error? I had looked awhile ago at resolving these somehow... but to no avail. :(

Reply:
Possibly, but it's worse to have 30000 other errors and miss important warnings from those. We'll catch a runtime error at runtime so at least they'll be caught somewhere

Reply:
Good point.. It is cumbersome to filter through all them.